example/Cargo.toml

```
@@ -0,0 +1,17 @@
+[package]
+name = "example"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+sdl2 = {version= "0.35.2", features = ["raw-window-handle", "static-link", "use-pkgconfig"] }
```

```suggestion
sdl2 = {version= "0.35.2", features = ["raw-window-handle"] }
```
Had to remove these to make it compile on MacOS